### DISCUSSION TITLE
Support for Ubuntu 24.04 LTS ?

### DISCUSSION THREAD

**Discussion by ayush6678** [nnstreamer/nnstreamer#4505] [Q&A] [answered]
Hello maintainers, NNstreamer works great on ubuntu 22.04. I just wanted to know when nnstreamer will support Ubuntu 24.04. Thanks!

**Comment by myungjoo** [comment, marked as answer]
It is working on Ubuntu 24.04; if you build NNStreamer with gcc/llvm, it's built and it runs flawlessly.

We just don't have PPA release ready for Ubuntu 24.04. It will be released when PyTorch-by-Ubuntu for 24.04 gets ready: https://code.launchpad.net/~nnstreamer/+recipe/pytorch-daily-noble
Ubuntu's PyTorch build is not yet ready for 24.04.

**Reply by ayush6678** [reply]
Yes, I was able to build nnstreamer from source but I was facing problems building nnstreamer-tensorflow and couldn't get tflite models to work with nnstreamer pipelines. I am stuck with this for quite a few days. Maybe I am missing something, I am still new working with nnstreamer.

**Comment by myungjoo** [comment]
uh.. what the hell. let's forget Ubuntu 24.04 PPA Pytorch support and keep it moving: #4641

**Reply by myungjoo** [reply]
way too late. but published.
